src/main/java/com/google/ehub/servlets/ProfileDatastore.java

```diff
@@ -0,0 +1,45 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import java.io.IOException;
+import java.io.PrintWriter;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**Manages the User Profiles stored in Datastore.**/
+public final class ProfileDatastore {
+  public static final String PROFILE_ITEM_KIND = "profile";
+  public static final String NAME_PROPERTY_KEY = "name";
+  public static final String EMAIL_PROPERTY_KEY = "email";
+  public static final String USERNAME_PROPERTY_KEY = "username";
+  public static final String BIO_PROPERTY_KEY = "bio";
+
+  DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
the datastore member variable should be made private and final since it is not supposed to get re-assigned.

Reply:
I thought that since datastore will have different items, it shouldn't be final

Reply:
final just means that you can't re-assign the object to another reference again. If you declare it final nothing would affect the fact that you are adding multiple objects,

Reply:
Oh okay, I will make it final